src/main/kotlin/controllers/worldRenderer/Renderer.kt

```diff
@@ -157,6 +159,9 @@ class Renderer(
         setFpsTarget(configOptions.fpsCap.value.get() ?: 0)
 
         configOptions.priorityRenderer.value.addListener(::priorityRendererPref::set)
+        configOptions.skyColor.value.addListener {
+            try { skyColor = Color.decode(it) } catch (e: NumberFormatException) {}
```

Comment:
Might also want to request a new frame here? i.e. allow it to update in real time in power saving mode